Add 18.5, 19, and modify 17.5

KunzitePhantasmagoria opened the pull request:
Add boxes for 18.5 and 19. Update 17.5 to include Yuuma and Easy mode for Nintendo Switch version.
Alignment for STB, DS, ISC, and VD are a little wacky but functional with inclusion of 18.5 (With Clear and 85 card options)

KunzitePhantasmagoria committed:
Add 18.5, 19, and modify 17.5

Yuuma Toutetsu and Easy mode for 17.5

doopu commented:
Just a little nitpick - tabs vs. spaces; this repo is using spaces instead of tabs. Otherwise, it looks like you've done a good job; I will check how it looks later today (or if you want to attach an imgur link or something, I'm happy to trust that too!)

Thanks for the contribution, people have been chasing me for these for a while!

KunzitePhantasmagoria commented:
It's been like 2 years since i've really coded anything but I really like using this tracker so I wanted to help out. As for the tabs v spaces thing, that might have just been my editor because I remember using tabs ^^'

I wasn't really sure where to put Black Market, so I moved the STB and other games up, but now the alignment is off and I'm not entirely sure how to fix that, here's an image, along with colours I chose which honestly were a little arbitrary. (Also I didn't mean to have them transparent)

I'm also unaware if the switch version of GI's Greedy Challenge is an endless mode or just a boss rush, so I left it out for now

![NewGames](https://github.com/doopu/1ccTracker/assets/142266173/9511da4f-87fd-4e2c-98b7-9f10633cdc26)
![NewGames (Easy)](https://github.com/doopu/1ccTracker/assets/142266173/7b35a900-23b2-4eea-8b4a-79d9614628ed)

doopu commented:
The original colours were chosen by Anonymous, so yeah, it's gonna be arbitrary. But thinking about it, how about a nice green for TH19? Like the one from: https://en.touhouwiki.net/wiki/File:Th19cover.png

As for the layout, can we shrink the gap between the extra header and TH19 by a little? There's a lot of dead space between the two now. The position of 100BM is great! It fills in some of the space we had there before.

One more change: now 100BM is there, there's no reason to push STB + the others down for Easy Mode. They can stay where they are regardless of Easy Mode. It was a bit weird that I moved them at all for easy mode to be honest...

Thanks for adding in Easy for GI, I could find NO information on whether or not that game had an easy mode, so I left it off.

KunzitePhantasmagoria commented:
I'm a little colourblind but I picked this clour from the font: 74, 124, 71.  I'll show a picture when i look at the following:

As for the gap I didn't intend for it to be so big, so I'll take a look at that in a moment.

With STB and the others, I couldn't get them to stay put with the toggle, but I didn't spend a lot of time trying to fix that, so I'll take a look at that as well and let you know if I figure it out.

Easy mode for GI wasn't available in the original, but included for the NS version. I'll be getting it someday and if I find out Greedy Challenge is trackable I'll add it then.

KunzitePhantasmagoria commented:
Okay, I fixed everything from above. Turns out I was looking at the wrong part when trying to get STB and others from moving when easy was toggled, so that was quick this time around. Any other changes you'd like me to make?

The IN box was misaligned for a moment but I just updated the image, so there wont be a problem with that

Also i misread one of your comments, I changed all the tabs I saw to spaces, I don't think I missed any

![progress(13)](https://github.com/doopu/1ccTracker/assets/142266173/d314f355-0ca2-4ca7-b10e-ee2f5d8570e9)
![progress(1)](https://github.com/doopu/1ccTracker/assets/142266173/5b194eb3-6e57-4ddb-ace3-bf7ed71d1c9f)

KunzitePhantasmagoria committed:
Fix alignment and pretty up the page a lil

doopu commented:
Looks great! Push the commit (unless you've already done that I'm failing Github), I'll give it a once over and we're good to go!

KunzitePhantasmagoria commented:
Okay! Pushed, and should be ready to merge.

doopu commented:
Merged! It's live now (might need a hard refresh) - thank you again for the contribution!

KunzitePhantasmagoria commented:
Thank you, and it's already working for me! I really wanted to see the new games especially with 19 having... well 19 characters and it's nice to keep track of it with something, and I figured rather than just request it, I wanted to see if I could help implement it too
